src/scenes/home/header/sideNav/sideNav.css

```diff
@@ -40,7 +40,7 @@
   margin-top: 80px;
   padding: 30px 5%;
   border-top: 1px solid #fafafa;
-  font-size: 2.5em;
+  font-size: xx-large;
```

Comment:
Everyone has a different screen size, and the `xx-large` allows for some wiggle room vs a hard set number.